tdrs-backend/tdpservice/parsers/schema_defs/tanf/t1.py

```diff
@@ -244,18 +252,23 @@
                               validators.oneOf([1, 2]),
                         ]),
                   Field(item="27", name='WAIVER_EVAL_CONTROL_GRPS', type='string', startIndex=113, endIndex=114,
-                        required=True, validators=[
-                              validators.or_validators(validators.matches('9'), validators.isEmpty()),
-                              validators.isAlphaNumeric(),
+                        can_be_empty=True, required=True, validators=[
```

Comment:
The below snippet in particular sticks out to me as not very readable and additionally -- could we utilize `required=False` as our flag for `can be empty`? Or is that too complicated/overloading.

Reply:
We can't use the `required` flag if we still want the field validators to run. The way I understand it is that not all STTs will space fill these fields so we still want the validators to run. If that is not the case and Im mistaken then we can get away with only using the `required` flag.